### PR TITLE
Drop unsupported python (<3.5) and django (< 2.2)

### LINKED ISSUE #213: Drop unsupported python (< 3.5) and Django (< 2.2)
This project is still tagged and tested with unsupported version of both django and python.

Oldest supported Django version is **`2.2`** branch, which supports python from **`3.5`** onward.

Also python 3.5 is EOL and is not supported by Django 3.x.

https://docs.djangoproject.com/en/3.2/faq/install/#what-python-version-can-i-use-with-django
https://www.python.org/downloads/

### PR DESCRIPTION
**Pull request by sevdog**
Resolve #213.

- Change `url` to `path`
- Use `@admin.register` instead of `admin.site.register`
- Use `__str__` instead of `__unicode__`
- Change Python2 safe imports and other Python2 elements (thanks to [`2to3`](https://docs.python.org/3/library/2to3.html))

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -156,7 +149,7 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     long_description=LONG_DESCRIPTION,
     platforms=['any'],
     classifiers=CLASSIFIERS,
-    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0", "django>1.10"],
+    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0", "django>=2.2"],
```
Comment (pcraciunoiu) [inline setup.py:152]: On line 150 above, it still has `python_requires='>=2.7'` - could you update that, as well?

**Review by pcraciunoiu — COMMENTED** [review]
LGTM just a small comment below +

The readme still says django 1.11, could you update the requirements? And while you're there, also specify python 3.5 or later?

Would you be interested in also updating the badge in the README from Travis.ci to GitHub Actions badge?

https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge

File: setup.py (hunk 1 above)
Comment (sevdog) [reply setup.py:152]: No problem, I missed it.

File: README.rst [hunk 2]
```diff
@@ -522,9 +522,9 @@ To create a release::
 
 .. |pypi| image:: https://badge.fury.io/py/django-ses.svg
     :target: http://badge.fury.io/py/django-ses
-.. |build| image:: https://travis-ci.com/django-ses/django-ses.svg?branch=master
+.. |build| image:: hhttps://github.com/django-ses/django-ses/actions/workflows/ci.yml/badge.svg
```
Comment (pcraciunoiu) [inline README.rst:525]:
```suggestion
.. |build| image:: https://github.com/django-ses/django-ses/actions/workflows/ci.yml/badge.svg
```

**Review by pcraciunoiu — APPROVED** [review]

File: django_ses/urls.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
-from django.conf.urls import url
+from django.conf.urls import path
```
Comment (mondaini) [inline django_ses/urls.py:1]: Hi. I believe the correct namespace path is `django.urls` 
Then the import should be `from django.urls import path`

Thank you for your hard work on this library.

Comment (mondaini) [reply django_ses/urls.py:1]: I'm getting the following exception after this release: 
```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 118, in inner_run
    self.check(display_num_errors=True)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 412, in check
    for pattern in self.url_patterns:
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 598, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 591, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/filipe/bairesdev/inBuildBE/inBuildBE/urls.py", line 66, in <module>
    path("admin/django-ses/", include("django_ses.urls")),
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django_ses/urls.py", line 1, in <module>
    from django.conf.urls import path
ImportError: cannot import name 'path' from 'django.conf.urls' (/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/conf/urls/__init__.py)

```

It resumes working after changing the import line.

Comment (pcraciunoiu) [reply django_ses/urls.py:1]: Thanks! Would you open a PR with the fix?

Comment (pcraciunoiu) [reply django_ses/urls.py:1]: I ended up pushing a fix 8bfd17fed03480453ec5756009abd804135dac99

Comment (pcraciunoiu) [reply django_ses/urls.py:1]: Try 2.1.1 https://pypi.org/project/django-ses/2.1.1/

**Pull request merged**